guide/principles/principles.md

```diff
@@ -0,0 +1,168 @@
+
+**Do**
+- Be open and transparent with how your product works
+- Build your product as open-source when possible
+- Let people verify, without having to trust
+- Explain what risks the user is taking on, and how best to mitigate them
+
+**Don't**
+- Make claims that are not explained or verifiable 
+
+---
+
+## Self-custody
+
+> Let users control their private keys, with no risk for seizure or freezing of funds
+
+Our existing mental models of access to digital services are usernames and passwords, controlled by a company that has custody of your funds and data. With everyone having direct access to the Bitcoin network without middlemen, there is no longer a need for people to delegate control of their funds. While it comes with greater responsibility, it is self-custody that enables the open financial system of peer-to-peer transactions.
+
+**Do**
+- Let users control their private keys and funds directly
+
+**Don't**
+- Custody funds for your users
+- Build products where the users' funds can be seized, or frozen
+
+---
+
+## Security
+
+> Provide appropriate and progressive security for all types of users
+
+Self-custody means that the end user is ultimately responsible for keeping their funds and private keys safe. They can only do that if we provide them with good, appropriate tools and awareness of best practices. It is therefore our responsibility as designers and makers to really take their security seriously and make Bitcoin products that are as secure as possible by default. We should strive for no loss of funds, regardless of whether this happens through negligence or theft. 
+
+Security is especially important when onboarding people new to Bitcoin where solutions that might be fine for advanced users might backfire for beginners. New users are likely to start by storing small amounts until they get comfortable with self-custody. A scheme suited for storing large amounts of funds is likely overkill and scary to a beginner, so the concept of progressive security is a good idea. This could let a user upgrade their security and private key management scheme as their savings grow. 
+
+A big part of security is also education and awareness, protecting users from bad actors and potentially their own bad behavior. Bad actors may already know what most users don’t, and be aware of new vulnerabilities. It is unrealistic to expect beginners to take in all the knowledge acquired by advanced users of several years in one go, for example while onboarding to a product. We should therefore consider how to continuously educate and level up user awareness of best practices and risks.
+
+Security can also be a feeling. A product that is polished, good looking, easy to use and transparently communicates how it works can help users feel more secure. Especialy compared to one with the exact same security measures that lack these qualities.
+
+
+**Do**
+- Take safeguarding of users funds seriously
```

Comment:
- Provide suitable private key management schemes for beginners

I feel like this one was already mentioned in the paragraphs above. Maybe you had something different in mind, what do you mean when you say `sustainable PKM`?

Reply:
The do's and don'ts sometimes repeat information from the paragraph in bullet points, I think that's OK.

Sustainable -> Suitable (meaning appropriate for beginners)